### ISSUE TITLE
Stylistic Sets not being applied correctly

### ISSUE THREAD

**Issue by xxKeefer** [github/mona-sans#83]
![image](https://github.com/user-attachments/assets/f97a8261-f3b2-4df5-b19a-8fd3b3d4bb0e)

As you can see in the CSS, `ss01` (Square diacritical marks) is off, `ss03` (Lowercase l with tail) is on, `ss05` (Double-storey a) is on and `ss07` (Round G) is off. But the the rendered text has none of those feature correctly applied.

I have tested on chromium (Brave and Edge) and on FireFox

**Comment by maulana-kurniawan** [comment]
Yeah, the stylic set not applied. Any maintainer please check this issue.

**Comment by GitHub-Creeper** [comment]
same issue here with also 0 being not 0 with a slash or hyphen

**Comment by tonylondon** [comment]
I am also experiencing this exact same issue, please could someone look into it? Thanks

**Comment by tonylondon** [comment]
I found that using the file MonaSansVF-Regular.woff contained in fonts/webfonts-variable works with all Stylistic Sets. I hope this helps someone.

**Comment by SableRaf** [comment]
> I found that using the file MonaSansVF-Regular.woff2 contained in fonts/webfonts-variable works with all Stylistic Sets. I hope this helps someone.

Thank you! I've been fighting with this for hours. 

> [!WARNING] 
> The Variable Font file `MonaSansVF-Regular.woff2` is not included in the release. It is at [`fonts/webfonts-variable`](https://github.com/github/mona-sans/tree/main/fonts/webfonts-variable).

**Comment by MerryMacMapface** [comment]
> Warning
> 
> The Variable Font file `MonaSansVF-Regular.woff2` is not included in the release. It is at [`fonts/webfonts-variable`](https://github.com/github/mona-sans/tree/main/fonts/webfonts-variable).

The mentioned file is not available anymore. Where can I get it?

**Comment by kylewaynebenson** [comment]
This was a documentation issue—we adjusted the stylistic alts between v1, 2, and 2+. I'll clarify in the readme. Thanks all!
